crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```diff
@@ -82,7 +83,8 @@ itertools = "0.12"
 workspace = true
 
 [features]
-default = ["tx2", "tx5"]
+
+default = ["tx5"]
```

Comment:
```suggestion
default = ["dep:tx5"]
```

But, also: do we want this to be an optional features? Kitsune doesn't work without it.

Reply:
What does `dep:` do?

Reply:
It distinguishes between features that are optional dependencies (`dep:`) and non-dependency features added by the crate dev.

Reply:
Actually, I removed all use of the tx5 feature in this PR, since there is no alternative. So I think the change should actually be:

```suggestion
default = []
```

Reply:
Perhaps instead we should just make the tx5 dep non-optional, and make the default feature list empty?

Reply:
That was my suggestion at the top of this thread.

Reply:
Ah, I get it now. Yes I'll make the dep non-optional